boutiques_descriptors/scrnabox_1.53.01.json

```diff
@@ -14,10 +14,11 @@
         {
             "name": "Working directory",
             "command-line-flag": "-d",
-            "value-key": "[DIR]",
+            "value-key": "[WORKING_DIR]",
             "type": "File",
-            "id": "dir",
-            "description": "Working directory (where all the outputs will be printed) it will created if it does not exist in you current directory",
+            "id": "working_dir",
+            "default-value": "scRNAbox_workdir",
+            "description": "The working directory where the output files will be saved.",
```

Comment:
I don't like that the description says this is an OUTPUT directory even though users select this as an INPUT. There shoudl be a clear explanation that it's both an input and an output